src/commands/check.js

```diff
@@ -68,7 +78,7 @@ export const handler = catchAsyncError(async opts => {
   const filteredWith = filter ? `filtered with ${strong(filter)} ` : '';
 
   console.log(
-    `Checking for outdated ${depsGroupsToCheckStr}dependencies ${filteredWith}for "${strong(packageFile)}"...`
+    `Checking for outdated ${depsGroupsToCheckStr}dependencies ${filteredWith}${opts.global ? '' : (`for "${strong(packageFile)}"`)}...`
```

Comment:
Can we show "Checking for outdated **global** dependencies..." when `--global` flag is set?